docs/MongoDB.md

```diff
@@ -35,13 +35,13 @@ describe('insert', () => {
   beforeAll(async () => {
     connection = await MongoClient.connect(globalThis.__MONGO_URI__, {
       useNewUrlParser: true,
+      useUnifiedTopology: true,
```

Comment:
example in docs is copied from https://github.com/facebook/jest/tree/main/examples/mongodb, please keep in sync